content/news/2024-05-14-SPLGAZAORDER.md

```diff
@@ -27,7 +27,7 @@ The vaguely worded update, which went into effect last week without the public f
 
 It has also heightened fears among pro-Palestinian and Muslim American groups that employees who express opinions that differ from the Democratic governor’s stance on the war could be unfairly punished.
 
-{{<embed/newsletter>}}
+<script src="https://www.spotlightpa.org/embed.js" async></script><div class="spl-newsletter" data-spl-embed-version="1" data-spl-src="https://www.spotlightpa.org/embeds/newsletter/"></div>
```

Comment:
Can we just apply `spl-newsletter` to a container in embed/newsletter? I don't want to stop using shortcodes for newsletters. It makes it possible to swap them out for links in the app, which is an important goal.